### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by hrsakai**
Use github actions for precommit job.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Node.js
+on: [pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```
Comment (massakam) [inline .github/workflows/nodejs.yml:6]: Is it not possible to use the `apachepulsar/pulsar-build` image directly?

Comment (okuryu) [reply .github/workflows/nodejs.yml:6]: You can probably do that with self-hosted runners.
https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners

Comment (hrsakai) [reply .github/workflows/nodejs.yml:6]: @massakam 
By default, it seems that only the virtual environments listed in the table on the following page can be used.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

@okuryu 
Thank you for your comment.
I will confirm self-hosted runners.

Comment (hrsakai) [reply .github/workflows/nodejs.yml:6]: I read the documents for self-hosted runners.
There are things to consider(security,server,etc), so we would like to consider self-hosted runners again in the future as needed.

**Review by nkurihar — APPROVED** [review]

**Review by massakam — APPROVED** [review]

**Pull request merged**